sources/procfs.go

```diff
@@ -320,6 +325,14 @@ func (s *lustreProcfsSource) generateClientMetricTemplates() error {
 			{"stats", "inode_permission_total", inodePermissionHelp, s.counterMetric},
 			{"xattr_cache", "xattr_cache_enabled", "Returns '1' if extended attribute cache is enabled", s.gaugeMetric},
 		},
+		"mdc/*": {
+			{"rpc_stats", "rpc_in_flight", rpcsInFlightHelp, s.gaugeMetric},
```

Comment:
So...I see that this metric is named differently than the 'rpcs_in_flight' value in osc/*. Maybe we just need to name these mdc_rpcs_in_flight and osc_rpcs_in_flight?